src/odemis/acq/fastem.py

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on 19 Apr 2021
+
+Copyright © 2021 Philip Winkler, Delmic
+
+This file is part of Odemis.
+
+Odemis is free software: you can redistribute it and/or modify it under the terms
+of the GNU General Public License version 2 as published by the Free Software
+Foundation.
+
+Odemis is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR
+PURPOSE. See the GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License along with
+Odemis. If not, see http://www.gnu.org/licenses/.
+"""
+from odemis import model
+
+
+class FastEMROA(object):
+    """ Representation of a FastEM acquisition region. """
+
+    def __init__(self, name, coordinates, calibration):
```

Comment:
Add a docstring with the type and meanings. In particular, define the coordinates (left,top,bottom,right)?

Calibration: maybe better call it "roc".